doodle.py

```diff
@@ -83,25 +121,15 @@ def dates_from_arg(args):
     end, relative = derelativize_date(parts[1], start)
     ndays = (end - start).days + 1  # Number of days to generate slots on
 
+    weekday = start.weekday()
```

Comment:
As commented above, I don't get why this is calculated only for the start date instead of for each date candidate.